src/Extension/CoreExtension.php

```diff
@@ -1286,6 +1285,8 @@ public static function capitalize(string $charset, $string): string
 
     /**
      * @internal
+     *
+     * to be removed in 4.0
```

Comment:
As it is internal, can't it be removed immediately ?

Reply:
It's used by the deprecated `MethodCallExpression` class, which is public.